articles/communication-services/concepts/troubleshooting-info.md

```diff
@@ -291,6 +291,7 @@ The Azure Communication Services SMS SDK uses the following error codes to help
 | 4006 | The Destination/To number is not reachable| Try re-sending the message at a later time |
 | 4007 | The Destination/To number has opted out of receiving messages from you| Mark the Destination/To number as opted out so that no further message attempts are made to the number|
 | 4008 | You have exceeded the maximum number of messages allowed for your profile| Ensure you are not exceeding the maximum number of messages allowed for your number or use queues to batch the messages |
+| 4009 | Message is rejected by Microsoft Entitlement System| Most often it happens if fradulent activity is detected. Please contact support for more details |
```

Comment:
Actually the error text we return to customer now contains Entitlement. i.e. "Message is rejected by Microsoft Entitlement System". And rejected by Entitlement is not necessary because of Fraud reasons. For Fraud detected reasons we have special code 4000, see above

Reply:
let's keep current code documented. @Prakulka for changing message from the system we can have a separate ticket